app/js/controllers/resultat.js

```diff
@@ -46,7 +46,7 @@ angular.module('ddsApp').controller('ResultatCtrl', function($scope, $rootScope,
     };
 
     $scope.shouldDisplayImpactInfo = function(partenaireId) {
-        return ! droitsDescription.partenairesLocaux[partenaireId].interactionWithNationalPrestationCalculated;
+        return droitsDescription.partenairesLocaux[partenaireId].interactionWithNationalPrestationCalculated !== false;
```

Comment:
Non, l'intention est bien que l'interaction soit un opt-in. Par défaut, on ne sait pas s'il y a des interactions. Ce n'est qu'après analyse juridique qu'on peut certifier qu'elles ont été prises en compte (y compris pour dire qu'il n'y en a aucune).